server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -283,7 +288,15 @@ public StreamObserver<AutoCompleteRequest> autoCompleteStream(
                         "from deephaven_internal.auto_completer import jedi_settings ; jedi_settings.set_scope(globals())");
                 settings[0] = (PyObject) scriptSession.getVariable("jedi_settings");
             } catch (Exception err) {
-                log.error().append("Error trying to enable jedi autocomplete").append(err).endl();
+                if (err instanceof QueryScope.MissingVariableException) {
+                    if (!ALREADY_WARNED_ABOUT_NO_AUTOCOMPLETE.getAndSet(true)) {
+                        log.error().append("Autocomplete package not found; disabling autocomplete.").endl();
+                        log.error().append("Add the jvm flag '-D").append(DISABLE_AUTOCOMPLETE_FLAG)
+                                .append("=true' to disable this message.").endl();
+                    }
+                } else {
+                    log.error().append("Error trying to enable jedi autocomplete").append(err).endl();
+                }
```

Comment:
I don't know if specifically checking against MissingVariableException is something we need to do - I say catch any exception and set the flag.

Additionally, I think mentioning `pip install deephaven-core[autocomplete]==<version>` as a potential fix would be helpful here.